### PR TITLE
Localized impact form in id

### PR DESCRIPTION
**Pull request by kelimuttu**

### CODE REVIEW

File: _includes/footer-nolang.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<footer class="site-footer">
```
Comment (MichaelKohler) [inline _includes/footer-nolang.html]: I think we don't need this. We can just disable the language itself in the dropdown in the footer. To do this, add "id" here: https://github.com/mozilla/activate.mozilla.community/blob/gh-pages/_config.yml#L24 and https://github.com/mozilla/activate.mozilla.community/blob/gh-pages/_config-dev.yml#L24

**Comment by kelimuttu** [comment]
I'm not sure if I'm doing it right. Because if I go to /howwasit, the dropdown still showing 2 options which all of them are EN. But when I click one of them, it leads to the id one. Can you check on your side @MichaelKohler ?

File: _includes/footer-nolang.html (hunk 1 above)
Comment (kelimuttu) [inline _includes/footer-nolang.html]: lang disabled and .html file has been deleted

**Comment by MichaelKohler** [comment]
That sounds interesting, I'll have a look 😯

**Comment by MichaelKohler** [comment]
The cause was that the "lang: en" property was set on _pages/id/impactform.md. I've created a PR against your version of the repository, to the correct branch. Once you merge that, it should show up here. https://github.com/kelimuttu/activate.mozilla.community/pull/1

That basically is a PR against your branch, when you merge it will update your branch that this PR here depends on, and then it will show up. Kind of a PR-inception. 🔨

File: _layouts/impactform.html [hunk 2]
```diff
@@ -1,3 +1,5 @@
+{% assign l = site.data.l10n.[page.lang] %}
```
Comment (MichaelKohler) [inline _layouts/impactform.html]: as the "l" variable is not used further down in this form, this can be removed as well.

**Comment by MichaelKohler** [comment]
Closing in favor of https://github.com/mozilla/activate.mozilla.community/pull/257

**Pull request closed without merging**